test/edu/washington/escience/myria/operator/DupElimTest.java

```diff
@@ -94,7 +94,8 @@
   @Test
   public void testDupElim() throws DbException {
     BatchTupleSource src = new BatchTupleSource(makeTestData());
-    StreamingStateWrapper dupElim = new StreamingStateWrapper(src, new DupElim());
+    // StreamingStateWrapper dupElim = new StreamingStateWrapper(src, new DupElim());
```

Comment:
Yeah, that change was just temporary to show this passes the unit tests, but I guess I'll just change it to `HashDupElim` since I assume that will be the new default.